api/proxy/servers/arbitrum_altda/handlers.go

```diff
@@ -0,0 +1,186 @@
+package arbitrum_altda
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+/*
+	This is a (hopefully) comprehensive handlers blue print for introducing a new ALT DA server type
+	that's compatible with Arbitrum's upcoming Custom DA spec.
+
+	TODO: Understand the best way to incorporate the ALT DA server into the
+	 	  existing eigenda-proxy E2E testing framework
+
+	TODO: Understand what fork management for our Arbitrum forks will look like; at a high level we need:
+			1. test E2E correctness of the nitro stack with EigenDA.
+			2. introduce missing key security checks that could impact the integration's L2 Beat assessment
+
+	TODO: Consider that we'll need access to the Arbitrum's ALT DA client for testing, we'll need to either
+	      import (likely infeasible) or vendor. vendoring might be the only option for this - we've talked about
+		  doing this on other code sections across the monorepo. There could be some way to leverage a programming
+		  agent to do detection for when vendor'd code should update to help smooth the maintenance burden introduced.
+
+	TODO: Method implementations:
+		[X] IsValidHeaderByte // trusted integration
+		[ ] Store // trusted integration
+		[ ] RecoverPayloadFromBatch // trusted integration
+		[ ] GenerateProof // secure integration
+		[ ] GenerateCertificateValidityProof // secure integration
+*/
+
+// Handlers defines the Arbitrum ALT DA server spec's JSON RPC methods
+// This method implementations should serve as a thin wrapper over the existing EigenDA manager construct
+// with translation mapping 503 (failover) and 418 (invalid_cert) status codes into error messages that
+// arbitrum nitro can understand to take actions preserving both rollup liveness and safety
+//
+// Some custom code / refactoring will likely be necessary for supporting the READPREIMAGE proof serialization logic
+type Handlers struct {
+	// TODO: Metrics support - makes sense to share metrics server between both rest and arbitrum alt da
+	//       servers. go-ethereum might suppprot
```

Comment:
incomplete sentence. but agree we could use a label in the metrics to specify which route the request was received on